stacker/commands/stacker/info.py

```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
```

Comment:
maybe including like:

- created date
- last changed date

would be useful and fit more with info?

haven't looked into whether or not we have access to that info